tools/integration_tests/util/operations/file_operations.go

```diff
@@ -295,6 +295,31 @@ func ReadChunkFromFile(filePath string, chunkSize int64, offset int64) (chunk []
 	return
 }
 
+// Write data of chunkSize in file at given offset.
+func WriteChunkSizeInFile(file *os.File, chunkSize int, offset int64) error {
```

Comment:
Nit: This function name is a bit unclear as it doesn't tell what it writes in the file. Could you rename this to something like `WriteRandomBytesToFileChunk` ?